<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReconstructDialog</class>
 <widget class="QDialog" name="ReconstructDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>258</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reconstruct</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="selectExportLabel">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>151</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Select Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectExportComboBox">
       <property name="toolTip">
        <string>Select exported keyframes to reconstruct</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="selectReconstructtoolLabel">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>151</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Select reconstruct tool</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectReconstructtoolComboBox">
       <property name="toolTip">
        <string>hint: add your tool in settings-&gt;manage reconstruction tools</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="startArgumentsLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Start arguments for reconstruct tool executable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="startArgumentsLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>gui</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="startArgumentsHint">
       <property name="styleSheet">
        <string notr="true">color: rgb(58, 58, 58); border-left: 6px solid  rgb(58, 58, 58); border-top-right-radius: 5px; border-bottom-right-radius: 5px; background-color: lightblue;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use &amp;quot;gui&amp;quot; and the checkbox below to start the COLMAP gui w/o a project file.&lt;/p&gt;&lt;p&gt;Use &amp;quot;automatic_reconstructor&amp;quot; to start an automatic reconstruction (the software will add needed start arguments itsself).&lt;/p&gt;&lt;p&gt;Use any other start arguments and the software will create a batch file to manually start COLMAP with those start arguments.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="createprojectfileCheckBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Create project file automatically (Colmap)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="cancelbutton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reconstructButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Reconstruct</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
